docs/superpowers/plans/2026-07-01-is-stranky-search.md

```
@@ -0,0 +1,693 @@
+
+```ts
+export interface PinnedPagesSlice {
+  pinnedPages: PinnedPage[];
+  loadPinnedPages: () => Promise<void>;
+  pinPage: (page: PinnedPage) => Promise<void>;
+  unpinPage: (id: string) => Promise<void>;
+  migratePinnedIds: (navPages: PageCategory[]) => Promise<void>;
+}
+```
+
+In the `AppState` type union (single line, search for `export type AppState =`), remove ` & PinnedPagesSlice` from the middle of the chain (it currently sits between `ErasmusSlice &` and `MenuSlice &`).
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)